pkg/build/cmd/publishgithub.go

```diff
@@ -41,9 +43,10 @@ var (
 	errReleaseNotFound = errors.New(`release not found, use "--create" to create the release`)
 )
 
-func PublishGithub(ctx *cli.Context) error {
+func PublishGithub(cctx *cli.Context) error {
```

Comment:
It looks good to me, the only nit I have is that this cctx variable is a little too similar to ctx could make it difficult to understand the code later, maybe it would be better to change it to `cliCtx` ?

Reply:
I usually prefer `c` for this variable.

Reply:
Personally I'd prefer going with cliCtx but since we already have `c` in various other places, let's move that discussion somewhere else 🙂 I'll look into how we can implement a linter around these conventions so that they are picked up automatically.